Toolkit/AppDeployToolkit/AppDeployToolkitMain.ps1

```diff
@@ -6657,25 +6624,20 @@ Function Show-WelcomePrompt {
 		$listBoxCloseApps.FormattingEnabled = $true
 		$listBoxCloseApps.HorizontalScrollbar = $true
 		$listBoxCloseApps.Name = 'listBoxCloseApps'
-		$System_Drawing_Size = New-Object -TypeName 'System.Drawing.Size'
-		$System_Drawing_Size.Height = 100
-		$System_Drawing_Size.Width = 300
+		$System_Drawing_Size = New-Object -TypeName 'System.Drawing.Size' -ArgumentList 400,100
```

Comment:
Width changed from 300 to 400?

Reply:
This was due to a lot of unused space around the process list and for the fact because I intent to change the code to display application name and file name in the list.